spec/services/orders/create_service_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe ::Orders::CreateService do
```

Comment:
1) `describe ::Orders::CreateService do`
2) Add failed context (when order is not saved)
3) Add system spec